src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
             await builder.DefinePersistence(settings);
 
             builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
+            builder.NotifyOnFailedMessage(failedMessage =>
```

Comment:
It's only SC that has it's own "default server"

On Tue, Dec 8, 2015 at 9:23 AM, Tim Bussmann <notifications@github.com>
wrote:

> In src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs
> <https://github.com/Particular/NServiceBus/pull/3175#discussion_r46925606>
> :
>
> > @@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
> >              await builder.DefinePersistence(settings);
> >
> >              builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
> > +            builder.NotifyOnFailedMessage(failedMessage =>
>
> I'm not sure that removing the feature and instead putting it on the
> DefaultServer configuration is such a good idea. What if a downstream
> repos implements it's own DefaultServer (I think most do)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3175/files#r46925606>.
>